server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const path = require('path');
+const axios = require('axios');
+const db = require('../database/index.js');
+
+const app = express();
+const PORT = 3002;
+
+// TODO: update once static client files are available
+// app.use(express.static())
+app.use(express.json());
+app.use(express.urlencoded());
+
+app.get('/api/menu/:id', (req, res) => {
+  const { id } = req.params;
+  res.send(id);
```

Comment:
Assuming this is test code for now. Maybe should have a todo flag?
```suggestion
  res.send(id); // TODO: query database and send back restaurant document
```